src/docs/index.ts

```diff
@@ -29,6 +30,8 @@ export enum DocsHomepageElementName {
   GITHUB = 'Github',
   GRANTS = 'Grants',
   IMPLEMENT_SWAP = 'Implement Swap',
+  NEGATIVE_SENTIMENT = 'Negative Sentiment',
+  POSITIVE_SENTIMENT = 'Positive Sentiment',
```

Comment:
Would it be better to separate these out into it's own Enum here? These aren't homepage sections but rather sentiment event details basically, so a `DocsSentiment` enum seems to be more fitting to me!